botorch/acquisition/hentropy_search.py

```diff
@@ -0,0 +1,282 @@
+#!/usr/bin/env python3
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+r"""
+This file has an implementation of Batch H-Entropy Search (qHES) via
+one-shot optimization as introduced in [Neiswanger2022]_.
+
+The authors adopt a generalized definition of entropy from past work
+in Bayesian decision theory, which proposes a family of decision-theoretic
+entropies parameterized by a problem-specific loss function and
+action set. Therefore, the method allow the development of a common
+acquisition optimization procedure, which applies generically
+to many members of this family (where each member is induced by a
+specific loss function and action set).
```

Comment:
Perhaps it would be nice to add a brief definition of what the action set is? From the user's perspective they will probably be content with knowing that an action represents some point in the input space!

Reply:
or a set of points for tasks such as top-k estimation!

Reply:
That's a good idea. I add a brief description of the action set to the doc string.